src/pages/RequestCallPage.js

```diff
@@ -106,13 +106,13 @@ class RequestCallPage extends Component {
             Growl.error(this.props.translate('requestCallPage.growlMessageNoPersonalPolicy'), 3000);
             return;
         }
-
+        const phoneNumber = this.state.phoneNumber.replace(/((?!\n)[()-\s\t])/g, '');
```

Comment:
The regex in this file has been repeated 4 times. It seems pretty useful as well so how about moving it to a lib file and calling it via a method like `getPhoneNumberWithoutSpecialChars` that would do the `replace` as well, or moving the regex to CONST. This way we can DRY up the code.

Reply:
Done